src/main/resources/application-prod.yml

```
@@ -60,4 +60,9 @@ logging:
     org.springframework.security: INFO
     org.springframework.security.oauth2: INFO
     org.springframework.web.client.RestTemplate: INFO
-    com.example.dnd_13th_9_be: INFO
\ No newline at end of file
+    com.example.dnd_13th_9_be: INFO
+
+server:
+  tomcat:
+    max-http-form-post-size: 100MB
+    max-swallow-size: -1
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)